.github/workflows/notify_codeowners.yml

```
@@ -18,6 +18,7 @@ jobs:
       - name: Drop a message for codeowners
         env:
           PR: ${{ steps.findPr.outputs.pr }}
+          GITHUB_BOT_TOKEN: ${{ secrets.BOT_TOKEN }}
```

```suggestion
          BOT_TOKEN: ${{ secrets.BOT_TOKEN }}
```

Or could change the os env variable that is called.

---

Fixed it